packages/react-select/src/Async.js

```diff
@@ -86,15 +86,17 @@ export const makeAsyncSelect = <C: {}>(
         });
       }
     }
-    UNSAFE_componentWillReceiveProps(nextProps: C & AsyncProps) {
+    componentDidUpdate(prevProps: C & AsyncProps) {
```

Comment:
This method is better replaced with `getDerivedStateFromProps`. The work to do that has already been done https://github.com/JedWatson/react-select/pull/4313, so we're likely to merge that one first.

Reply:
I agree. I removed my changes regarding `Async` component in this PR.